google-cloud-bigquery/acceptance/bigquery/bigquery_test.rb

```diff
@@ -15,7 +15,7 @@
 require "bigquery_helper"
 
 describe Google::Cloud::Bigquery, :bigquery do
-  let(:publicdata_query) { "SELECT url FROM [publicdata:samples.github_nested] LIMIT 100" }
+  let(:publicdata_query) { "SELECT url FROM publicdata.samples.github_nested LIMIT 100" }
```

Comment:
Yes, but since the project doesn't have any `-` characters we don't need to ticks. Its good to have coverage both ways. :)